File: kantilever-case3/src/FrontendService/FrontendService/ClientApp/src/app/models/artikel.ts
export interface Artikel {
    id: number;
    naam: string;
    prijs: number;
    prijsInclBtw: number;
    beschikbaarAantal: number;
    afbeeldingUrl: string;
    beschrijving: string;
    categorie: string;
}
export interface ArtikelDetails {
    id: number;
    naam: string;
    prijs: number;
    prijsInclBtw: number;
    beschikbaarAantal: number;
    afbeeldingUrl: string;
    beschrijving: string;
    leverbaarVanaf: Date;
    leverbaarTot: Date;
    leverancierCode: string;
    leverancier: string;
    categorie: string;
    subCategorie: string;
}

File: kantilever-case3/src/FrontendService/FrontendService/Models/Artikel.cs
using System;
using FrontendService.Constants;

namespace FrontendService.Models
{
    public class Artikel
    {
        public long Id { get; set; }
        public long Artikelnummer { get; set; }
        public string Naam { get; set; }
        public string Beschrijving { get; set; }
        public decimal Prijs { get; set; }
        public decimal PrijsInclBtw =>
            Math.Round(Prijs * SystemVariables.BtwMultiplier, SystemVariables.DefaultAmountOfDecimals);

        public string AfbeeldingUrl { get; set; }
        public DateTime? LeverbaarVanaf { get; set; }
        public DateTime? LeverbaarTot { get; set; }
        public string Leveranciercode { get; set; }
        public string Leverancier { get; set; }
        public string Categorie { get; set; }
        public string SubCategorie { get; set; }
        public int Voorraad { get; set; }

    }
}
